package.json

```diff
@@ -82,11 +85,15 @@
     "@langchain/openai": "0.4.4",
     "@octokit/rest": "21.1.1",
     "@slack/bolt": "4.2.0",
+    "@slack/web-api": "^7.8.0",
+    "axios": "1.7.9",
     "busboy": "1.6.0",
+    "csv-parse": "5.6.0",
```

Comment:
can we use the `csv` dependency as we do in other places or is there a specific need for separate parse and write deps?